src/main/scala/org/antipathy/mvn_scalafmt/SourcesBuilder.scala

```diff
@@ -0,0 +1,54 @@
+package org.antipathy.mvn_scalafmt
```

Comment:
compared to #149: moved the bulk of the logic from `FormatMojo.getSources` to this source, along with unit tests.